cmd/juju/controller/destroy.go

```diff
@@ -184,9 +184,17 @@ func (c *destroyCommand) Run(ctx *cmd.Context) error {
 		return errors.Trace(err)
 	}
 	store := c.ClientStore()
-	if !c.assumeYes {
+
+	skipConfirm, skipErr := jujucmd.CheckSkipConfirmEnvVar()
```

Comment:
I've seen it done before where options are defaulted to env vars by setting the struct attribute in the command.Init function. Such as here:
https://github.com/juju/juju/blob/ecc18e520bac2276a3e1d12260c4b0ec71bf3f12/cmd/juju/commands/bootstrap.go#L408-L410 

I think I would prefer this, as you could place it in destroyCommandBase and de-clutter Run

Reply:
Ok, fixed